docs/jasmine-upgrade.md

```diff
@@ -0,0 +1,76 @@
+## Upgrading from Jasmine 1.3 to 2.0
+
+First, please read the official upgrade document: http://jasmine.github.io/2.0/upgrading.html
+
+#### Focused specs
```

Comment:
Add a section on what you actually have to change in your Protractor configuration file.